### PR TITLE
feat: upgrade astro to version 3

### PR DESCRIPTION
**Pull request by javleds**
Changelog:
- Upgrade dependencies to run over astro v3.
- Remove experimental features (robots working as expected on new astro versions).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,8 +43,9 @@
     "eslint-plugin-unused-imports": "^2.0.0",
     "husky": "^8.0.1",
     "lint-staged": "^12.5.0",
+    "path-browserify": "^1.0.1",
```
Comment (ixartz) [inline package.json:46]: Why path-browserify is needed?

File: package.json [hunk 2]
```diff
@@ -56,5 +58,5 @@
   "volta": {
     "node": "16.15.1"
   },
-  "author": "Ixartz (https://github.com/ixartz)"
+  "author": "Javier Ledezma <juls0593@gmail.com>"
```
Comment (ixartz) [inline package.json]: Could you revert the change here?

**Review by ixartz — CHANGES_REQUESTED** [review]
Just a few comments. Otherwise, it looks good to me.

Thank you for making the PR.

File: package.json (hunk 2 above)
Comment (javleds) [reply package.json]: My apologies, we will revert this one.

File: package.json (hunk 1 above)
Comment (javleds) [reply package.json:46]: `path-browserify` is required for the `rss` plugin, please see:

Screenshot 2023-10-13 at 10 37 20

Comment (ixartz) [reply package.json:46]: Ok perfect, thank you

**Comment by ixartz** [comment]
Thank you for making the change, it looks good to me, can I merge?

**Comment by javleds** [comment]
> Thank you for making the change, it looks good to me, can I merge?

Hi @ixartz , for sure PR is finished now. Thank you.

**Review by ixartz — APPROVED** [review]

**Comment by ixartz** [comment]
@javleds Thank you for making this PR

**Pull request merged**